lib/circuit.js

```diff
@@ -582,7 +580,8 @@ class CircuitBreaker extends EventEmitter {
       const err = buildError('The circuit has been shutdown.', 'ESHUTDOWN');
       return Promise.reject(err);
     }
-    const args = Array.prototype.slice.call(rest);
+
+    const args = rest.slice();
```

Comment:
rest arguments become [] when no arguments provided, so Array.prototype.slice is guarding nothing.

Reply:
Sounds about right